Sources/Plasma/PubUtilLib/plNetGameLib/Intern.h

```diff
@@ -377,6 +377,26 @@ void NetTransCancelByConnId (unsigned connId, ENetError error);
 void NetTransCancelAll (ENetError error);
 void NetTransUpdate ();
 
+template<typename T, typename = void>
+struct HasTransId : std::false_type { };
+
+template<typename T>
+struct HasTransId<T, decltype((void)T::transId, void())> : std::true_type { };
```

Comment:
This is what C++17's `void_t` is for :)

```suggestion
struct HasTransId<T, std::void_t<decltype(&T::transId)>> : std::true_type { };
```

Reply:
Cool! I'm still trying to get the hang of SFINAE and all these little helpers.